Variations/ALKS_Scenario_4.1_3_SwervingSideVehicle_Variation.xosc

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<OpenSCENARIO>
+  <FileHeader revMajor="1" revMinor="1" date="2021-07-06T10:00:00" description="Variation for ALKS Scenario 4.1_3 Swerving Side Vehicle" author="BMW AG" />
+  <ParameterValueDistribution>
+    <ScenarioFile filepath="../Scenarios/ALKS_Scenario_4.1_3_SwervingSideVehicle_TEMPLATE.xosc" />
+    <Deterministic>
+      <DeterministicSingleParameterDistribution parameterName="Ego_InitSpeed_Ve0_kph">
```

Comment:
How to validate that a parameter does exists in the other file?

Reply:
This would be a semantic validation (not a syntactic validation against the schema). It's similar to checking, if a vehicle model, which is referred to in a scenario also exists in the catalog. Semantic checks are possible with the OSC API checker.